### ISSUE TITLE
Snapshots are uploaded with timestamp

### ISSUE THREAD

**Issue by raboof** [akka/akka-grpc#1050]
https://travis-ci.org/github/akka/akka-grpc/jobs/705323771 uploads snapshots as `1.0.0-17-3dcade13-20200706-0920` - apparently the directory has become dirty.

**Comment by raboof** [comment]
akka-grpc-runtime_2.12 is still fine but akka-grpc-runtime_2.13 was published with the timestamp

**Comment by ignasi35** [comment]
So this was a side-effect of #1049? How did that happen? (we check the scalafmt on PRs 🤔)

**Comment by raboof** [comment]
I guess 2 #1049 was merged in parallel with another PR
